### PR TITLE
[DOC] Add example of combining kernels and CBMA estimators

### PR DESCRIPTION
**Pull request by koudyk**
Refers to (but I'm not sure if it closes) #344 .

Changes proposed in this pull request:

I'm adding an example that shows how to use different kernels with the CBMA estimators. To do this, I basically copied the [plot_cmba.py](https://github.com/koudyk/NiMARE/blob/master/examples/02_meta-analyses/plot_cbma.py) file, and then for each estimator I loop through the different kernels and visualize the results.

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -68,6 +68,11 @@
       "affiliation": "Montreal Neurological Institute, McGill University",
       "orcid": "0000-0003-0556-0763"
     },
+    {
+      "name": "Oudyk, Kendra M.",
+      "affiliation": "Montreal Neurological Institute, McGill University",
+      "orcid": "0000-0003-4087-5402"
+    }
```
Comment (tsalo) [inline .zenodo.json]:
```suggestion
    },
```

File: examples/02_meta-analyses/plot_cbma_combine_estimators_and_kernels.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        draw_cross=False,
+        cmap="RdBu_r",
+        title="KDA estimator with %s" % kt_name,
+    )
+
+###############################################################################
+# ALE
+# --------------------------------------------------
+for kt_name, kt in kernel_transformers.items():
+    ale = nimare.meta.ale.ALE()
```
Comment (tsalo) [inline examples/02_meta-analyses/plot_cbma_combine_estimators_and_kernels.py:111]:
```suggestion
    ale = nimare.meta.ale.ALE(kernel_transformer=kt)
```

**Comment by tsalo** [comment]
It looks like we've found our first unexpected behavior! ALE doesn't seem to work with at least one of the other kernels.

**Comment by koudyk** [comment]
> It looks like we've found our first unexpected behavior! ALE doesn't seem to work with at least one of the other kernels.

 🎉
I was actually surprised that it worked when I first wrote it, but now I see that was because the code wasn't actually changing the kernel (thanks for catching that!).

**Comment by tsalo** [comment]
I think the best next step is to write in try/except statements so that the example will (1) continue past the broken kernels and (2) let us know that the kernels don't work. We can dig into why they don't work for ALE (and whether that's something we want to change) in future work.

**Comment by tsalo** [comment]
@koudyk Will you be able to add in the try/except statements? After that, I think I'll be able to review and merge pretty quickly.

**Comment by koudyk** [comment]
Yes @tsalo, I can try the try/except statements!

**Comment by tsalo** [comment]
Hi @koudyk, is there anything I can do to help get this over the finish line?

**Comment by koudyk** [comment]
> Hi @koudyk, is there anything I can do to help get this over the finish line?

Sorry about the delay! I felt some inertia because I haven't used try/except statements very often before. I just committed something; is it what you were envisioning?

File: examples/02_meta-analyses/plot_cbma_combine_estimators_and_kernels.py [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+            % kt_name
+        )
+
+###############################################################################
+# ALE
+# --------------------------------------------------
+
+
+for kt_name, kt in kernel_transformers.items():
+<<<<<<< HEAD
```
Comment (tsalo) [inline examples/02_meta-analyses/plot_cbma_combine_estimators_and_kernels.py:135]: It looks like there's a merge conflict that got committed without being resolved.

File: examples/02_meta-analyses/plot_cbma_combine_estimators_and_kernels.py [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    )
+    cres = corr.transform(ale.results)
+    plot_stat_map(
+        cres.get_map("logp_level-cluster_corr-FWE_method-montecarlo"),
+        cut_coords=[0, 0, -8],
+        draw_cross=False,
+        cmap="RdBu_r",
+        title="ALE estimator with %s" % kt_name,
+    )
+>>>>>>> 7a0d6866d69ea42c6529a9f59e10c1a47410d626
```
Comment (tsalo) [inline examples/02_meta-analyses/plot_cbma_combine_estimators_and_kernels.py:172]: I think this is the outdated part of the conflict.

**Review by tsalo — CHANGES_REQUESTED** [review]
This looks great! Just a small merge conflict that might not have been resolved.

File: examples/02_meta-analyses/plot_cbma_combine_estimators_and_kernels.py (hunk 3 above)
Comment (koudyk) [reply examples/02_meta-analyses/plot_cbma_combine_estimators_and_kernels.py]: You're right, sorry I didn't catch that! I think I fixed it now.

**Comment by tsalo** [comment]
I think we can mark this ready for review.

**Comment by koudyk** [comment]
> I think we can mark this ready for review.

Should I do that?

**Comment by tsalo** [comment]
Yes, please do.

**Comment by tsalo** [comment]
It looks like adding Dorota's info to the Zenodo file a few minutes ago introduced a merge conflict. Would you mind resolving that? I'll look at the code with fresh eyes tomorrow and probably merge then.

**Comment by koudyk** [comment]
I think I solved the merge conflict, but I can't tell

**Review by tsalo — APPROVED** [review]
Everything looks good to me. I think the failed CodeCov upload is the product of reopening the PR without a commit afterwards. Just a weird bug in the tool. Since I don't think it's going to fix itself, and this PR doesn't touch the core code, I'm going to merge as-is. Thank you for your contribution!

**Pull request merged**